os/kernel/Kconfig

```diff
@@ -1041,3 +1041,36 @@ config PTHREAD_STACK_DEFAULT
 		Default pthread stack size
 endmenu # Stack size information
 
+menu "Lockup detector"
+
+config SOFT_LOCKUP_DETECT
+	bool "Monitor the system for soft lockup"
+	default n
+	depends on !DISABLE_SIGNALS
+	select SCHED_WORKQUEUE
+	select SOFT_LOCK_MONITOR_PERIOD
+	select SLMWORKPERIOD
```

Comment:
Selections of SOFT_LOCK_MONITOR_PERIOD and SLMWORKPERIOD are not needed. Because SOFT_LOCKUP_DETECT is enabled, they will be enabled automatically.

Reply:
Ok Done